webcompat/views.py

```diff
@@ -73,10 +73,15 @@ def format_date(datestring):
 @app.route('/login')
 def login():
     if session.get('user_id', None) is None:
-        # manually set the referer so we know where to come back to
-        # when we return from GitHub
-        set_referer(request)
-        return github.authorize('public_repo')
+        if app.config['TESTING']:
+            session['username'] = 'testuser'
+            session['avatar_url'] = '/test-files/fixtures/avatar.png?'
+            return authorized()
+        else:
+            # manually set the referer so we know where to come back to
+            # when we return from GitHub
+            set_referer(request)
```

Comment:
I wonder if we'll want to do this for the mocked case, but let's merge and see if anything blows up. :)